### DISCUSSION TITLE
Issue with GitHub Student Developer Pack Benefits After Approval

### DISCUSSION THREAD

**Discussion by Suryanshu-Verma** [community/community#143078] [GitHub Education] [answered]
### Hi everyone,

Dear GitHub Education Support Team, 

I am writing to report an issue with accessing my GitHub Student Developer Pack benefits. Although my application for the Student Developer Pack was approved, I have yet to receive any of the education benefits associated with it.

GitHub Username: Suryanshu-Verma

Approval Date: 27/10/2024
Verification Method: I Provided  College ID 

Since receiving the approval notification, I have checked my GitHub account multiple times and ensured that my account settings are up to date. Unfortunately, the education benefits have not been activated.

I would greatly appreciate any assistance in resolving this issue or any steps I might need to take to access the Student Developer Pack benefits.

Thank you for your attention to this matter.

Best regards,
Suryanshu Verma
Suryanshu-Verma

**Comment by davevad93** [comment, marked as answer]
Hi @Suryanshu-Verma ,

What's the status of your application?

If the label is green and states "`Congrats! Your request for @Suryanshu-Verma was approved and your benefits will be available soon.` that means you have to wait. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

If the label is purple and states "`Congrats! Your request for @Suryanshu-Verma was approved and your benefits are now available.`", that means you should have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select one of the available offers, for example [Copilot](https://education.github.com/pack/redeem/copilot-student), if you can redeem it you are fine, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

For further reference, take a look at the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352):

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

And also the following discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268
